orttraining/orttraining/python/training/api/module.py

```diff
@@ -81,3 +83,33 @@ def save_checkpoint(self, ckpt_uri):
         """
         # TODO : move this out of Module Class.
         self._model.save_checkpoint(ckpt_uri)
+
+    # This function will change when the parameters will be exposed.
+    def get_contiguous_parameters(self, trainable_only: bool = False) -> OrtValue:
+        """
+        Returns contiguous parameters object.
+        """
+        size = self._model.get_parameters_size(trainable_only)
+        parameters = OrtValue.ortvalue_from_shape_and_type(
+            [
+                size,
+            ],
+            np.float32,
+            "cpu",
+            0,
+        )._ortvalue
+        self._model.copy_parameters_to_buffer(parameters)
+
+        return parameters
+
+    def get_parameters_size(self, trainable_only: bool = False) -> int:
```

Comment:
Do we need to expose this function here if `get_contiguous_parameters` is already allocating the buffer for the user? When would the user need this?

Reply:
This one is needed in the Federated learning scenarios where getting size is needed

Reply:
When would the user need to get the size? I think it is fine, just wanted to understand the context behind this.